app/script.js

```diff
@@ -0,0 +1,75 @@
+var information	 = [ "Do Better", "Try Harder", "You Got It", "Dont Stop Yet", "Awesome" ];
+
+function randomString(item) {
+var randomIndex = item[Math.floor(Math.random() * item.length)];
+return randomIndex;
+}
+
+/*	$('button').on('click', randomString());
+})
+
+var randomIndex = Math.floor(Math.random()*information.length);function randomString(words) {
+
+return words[Math.floor(Math.random() * words.length)];
+}
+*/
+/*$(function(){
+	$("button").click(function(){
+		$.get("http://localhost:3000/quote", function(response){
+		$("#ajax-text").text(response);
+
+		});
+	});
+});*/
+$(function() {
+	$(document).ready(function() {
+		$.get("http://localhost:3000/favorite", function(response) {
```

Comment:
if you change this get to just "favorite", it'll work on heroku. Right now, it just works locally. Check out the syntax in your other get/post requests and use that.